core/package.json

```
@@ -6,13 +6,15 @@
     "start": "../node_modules/concurrently/bin/concurrently.js \"npm run buildConfig:watch\" \"npm run serve\"",
     "serve": "sirv src -s -D -q -p 8080",
     "start:kyma": "npm run start",
-    "build": "npm run buildConfig",
+    "build": "npm run copy-luigi && npm run buildConfig",
```

```suggestion
    "build": "npm run buildConfig",
```
`copy-luigi` is called by `buildConfig` already.